### ISSUE TITLE
Datatable chokes on fast scrolling

### ISSUE THREAD

**Issue by k8n** [webix-hub/tracker#308]
I have a datatable set up with dynamic loading on a 10K record dataset. I can consistently reproduce this issue where if I scroll through the data set fast, the dynamic loader will hang.

I don't have the time to set up an example, I can however point out the issue in code, which will be apparent. I traced the bug to lines 11074 ... 11085 of webix_debug.js (4.3.8). That's end of '_feed_common' and the '_feed_callback' methods.

In '_feed_common', this.load(...) is called only when (this._feed_last.url !== url). However, if this.load(...) is not called,  this._load_count is not reset to false as it should be, which causes all subsequent data requests to be "ignored" (in '_feed', line 11017 ) thus breaking the component.

Thanks!

**Comment by k8n** [comment]
Here is my imperfect monkey patch:

```javascript
    // monkey patch for the webix.DataLoader.prototype._feed issue
    const original_feed = webix.DataLoader.prototype._feed
    webix.DataLoader.prototype._feed = function() {
		// if ajax queue is empty, but load_count is true-ish, break the lock
		if (this._load_count && !this._ajax_queue.length) {
			// breaking the lock
			this._load_count = false
		}
		return original_feed.apply(this, arguments)
	}
```

**Comment by mkozhukh** [comment]
Thanks for reporting the issue
Was fixed in webix 5.0
